### PR TITLE
Day6 assignment

### PR DESCRIPTION
**Pull request by rasika260896**

### CODE REVIEW

**Review by shreyagoyal30 — APPROVED** [review]

File: Day6-Assignment/src/App.js [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+        "akshaymarch7",
+        "karanmehta",
+    ]
+    useEffect(()=>{
+        userData();
+    },[]);
+
+ /*function to fetch github api */
+
+    async function userData(){
```
Comment (ankita1696) [inline Day6-Assignment/src/App.js:23]: can have this in a common component.

File: Day6-Assignment/src/Components/SearchBar.js [hunk 2]
```diff
@@ -0,0 +1,41 @@
+import React from 'react'
+import {useState} from 'react'
+
+/**
+	 * @description: function is called on Submit of the form, This function describes the functionality of Search
```
Comment (ankita1696) [inline Day6-Assignment/src/Components/SearchBar.js:5]: nicely added comments everywhere.

File: Day6-Assignment/src/Components/SearchBar.js [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+	 * @param1: searchText : entered search text
+     * @param2: user:props passed to the function to provide user data
+*/
+
+const searchMemeber = (searchText,user) =>{
+ return user.filter((item)=>{ return item.name.toLowerCase().includes(searchText.toLowerCase())})
```
Comment (ankita1696) [inline Day6-Assignment/src/Components/SearchBar.js:11]: should use option chaining as sometimes name property is not provided.

Comment (rasika260896) [reply Day6-Assignment/src/Components/SearchBar.js:11]: ok will incorporate this from next Assignment

**Pull request merged**